Rmarkdown/Multiple_Ascending_Dose_PKPD_continuous.Rmd

```diff
@@ -415,31 +306,25 @@ In a hysteresis plot, you want to determine whether the path is curving, and if
 In the plots below, most of the hysteresis paths follow a counter clockwise turn, with most arrows pointing up and to the right or down and to the left, indicating the effect increases in a delayed manner with increasing concentration.
 
 ```{r, cache = TRUE, fig.width = 8, fig.height = 15, warning = FALSE, message= FALSE}
-
 data_to_plot <- list()
-data_to_plot[[1]] <- my.data[my.data$CMT==2&my.data$TRTACT!="Placebo",c("TIME","LIDV","ID","TRTACT")]
+data_to_plot[[1]] <- pk_data[pk_data$TRTACT_low2high!="Placebo",c("TIME","LIDV","ID","TRTACT_low2high")]
 data_to_plot[[1]]$Concentration <- data_to_plot[[1]]$LIDV
 data_to_plot[[1]]$LIDV <- NULL
 
-data_to_plot[[2]] <- my.data[my.data$CMT==3,c("TIME","LIDV","ID","TRTACT")]
+data_to_plot[[2]] <- pd_data[pd_data$CMT==3,c("TIME","LIDV","ID","TRTACT_low2high")]
 data_to_plot[[2]]$Response <- data_to_plot[[2]]$LIDV
 data_to_plot[[2]]$LIDV <- NULL
 
-data_to_plot2 <- merge(data_to_plot[[1]],data_to_plot[[2]],by=c("TIME","ID","TRTACT"))
-
-data_to_plot2$TRTACT <- factor(data_to_plot2$TRTACT, 
-                                   levels = rev(levels(data_to_plot2$TRTACT)))
-
+data_to_plot2 <- merge(data_to_plot[[1]],data_to_plot[[2]],by=c("TIME","ID","TRTACT_low2high"))
 data_to_plot2<- data_to_plot2[order(data_to_plot2$TIME),]
 
-gg <- ggplot(data = data_to_plot2, aes(x=Concentration,y=Response, color=TIME))+theme_bw()
-gg <- gg + AnnotateStatus(draft.flag,fontsize=3)
+gg <- ggplot(data = data_to_plot2, aes(x=Concentration,y=Response, color=TIME))
```

Comment:
lines 309 through 321 can be deleted and replaced with:
gg <- ggplot(data = pkpd_data_wide, aes(x = CONC, y = RESPONSE, color = TIME)